.github/workflows/toc.yml

```
@@ -5,6 +5,6 @@ jobs:
     name: TOC Generator
     runs-on: ubuntu-latest
     steps:
-      - uses: technote-space/toc-generator@v4.2.1
+      - uses: technote-space/toc-generator@v4.2.2
```

```suggestion
      - uses: technote-space/toc-generator@v4
```
Let's pin the version to reduce the noise